docs/nginx.md

```diff
@@ -33,6 +33,24 @@ While this model gives you the ability to use whatever authentication backend yo
 
 Furthermore, introducing an extra http layer in your communication pipeline will make it more complex to deploy, maintain, and debug, and will possibly create issues.
```

Comment:
Could you add a blurb here about making sure the extra complexity is required before setting on such a setup?